SEED-platform/seed
# 2831 fix/delete analysis with property

#### Any background context you want to provide?
see ticket

#### What's this PR do?
Gives a use the option to also delete associated analyses when deleting a property.  Deleting 1:1 analyses is checked by default, whereas you must opt in to delete batch analyses.

#### How should this be manually tested?
Add some properties, run some analyses (including batches), delete used properties, ensure analyses are gone.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2831

#### Screenshots (if appropriate)
There are two properties and two analyses, one single and one batch.  Attempting to delete the property in both results in:
<img width="609" alt="Screen Shot 2022-03-15 at 8 25 55 PM" src="https://user-images.githubusercontent.com/564231/158505056-45fcdf88-786c-487e-a2c1-4224aa3111d7.png">
Attempting to delete the property just in the batch results in:
<img width="613" alt="Screen Shot 2022-03-15 at 8 34 41 PM" src="https://user-images.githubusercontent.com/564231/158505320-05625530-01a5-4cfa-9ad5-78ac738f3f17.png">
